apps/system/js/app_chrome.js

```diff
@@ -791,6 +791,12 @@
     };
 
   AppChrome.prototype.handleScrollAreaChanged = function(evt) {
+    // Make sure the scroll-area-changed is coming from the right element.
+    if (evt && (!this.app || !this.app.browser ||
```

Comment:
this `!this.app` is probably overcautious since the `AppChrome` constructor expects it as a param.